docs/maintenance.rst

```diff
@@ -0,0 +1,25 @@
+Maintenance
+===========
+
+.. note::
+
+    This page is only relevant for maintainers of :py:mod:`battleship_mp`.
+
+Releases
+--------
+
+The package follows `Semantic Versioning`_
+and is automatically published to PyPI using GitHub actions.
+To trigger a release:
+
+1. Commit a new version
+    - Adjust and commit the ``version`` in ``pyproject.toml``
+    - Create a git tag for on commit (e.g. via ``git tag -a "v1.1.2" -m "description"``)
```

Comment:
Is this actually needed? I think create a new `GitHub release` does it automatically. However, it does not hurt neither.

Reply:
This should be possible via the GH UI as well. I'll leave it as-is for now but may add a note later.